src/components/navigation/TopbarNotifications.tsx

```diff
@@ -66,31 +68,42 @@ const NotificationItem = ({ notification }: NotificationItemProps) => {
 
   const Icon = notification.read ? NotificationReadIcon : NotificationUnreadIcon;
 
+  const linkOnClick = () => {
+    event('open-notification-link', 'notifications', `Opened notification link: ${notification.link}`);
+    setShowNotifications((prev) => !prev);
```

Comment:
Sett denne til `false` ettersom den alltid skal bli false ved klikk på link.

```suggestion
    setShowNotifications(false);
```